samples/drivers/video/sw_pipeline/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.20.0)
+find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
+project(usb_video)
```

Comment:
should match name of sample

Reply:
I must have forgotten to change it. Adjusted thank you.